Assets/MixedRealityToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -3,12 +3,10 @@
 
 using MixedRealityToolkit.Common;
 using UnityEngine;
-
-#if UNITY_WSA && UNITY_2017_2_OR_NEWER
+using System;
 using System.Collections.Generic;
 using UnityEngine.XR;
 using UnityEngine.XR.WSA;
```

Comment:
@KevinKennedy anything using `UnityEngine.XR.WSA` needs the ` UNITY_WSA` definition.

Reply:
See earlier comment as to why